internal/api/front/perses/src/app/project/dashboard/ngxcharts.model.ts

```diff
@@ -0,0 +1,22 @@
+// Copyright 2021 Amadeus s.a.s
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+export interface NgxPoint {
```

Comment:
Ngx means angular extension for me, if you want to extend nx-chart library (this is what I understand) you'll probably have better to name it NgxChartPoint and NgxChartLine ?

Reply:
Some comments to explain what is the purpose of these interfaces would be cool.